packages/paste-core/components/corner-ornament/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import { render } from "@testing-library/react";
+import { Avatar } from "@twilio-paste/avatar";
+import { Box } from "@twilio-paste/box";
+import { LogoTwilioIcon } from "@twilio-paste/icons/esm/LogoTwilioIcon";
+import { IconSizeOptions } from "@twilio-paste/style-props";
+import * as React from "react";
+
+import {
+  CornerOrnament,
+  CornerOrnamentContainer,
+  CornerOrnamentElement,
+  CornerOrnamentPosition,
+  CornerOrnamentType,
+} from "../src";
+
+const ExampleCornerOrnament: React.FC<{
+  size?: IconSizeOptions;
+  position?: CornerOrnamentPosition;
+  type?: CornerOrnamentType;
+}> = ({ size, type }) => (
+  <CornerOrnamentContainer
+    data-testid="cornerOrnamentContainer"
+    cornerOrnamentType={type || "dot"}
+    size={size || "sizeIcon50"}
+  >
+    <CornerOrnamentElement data-testid="cornerOrnamentElement">
+      <Avatar data-testid="ornament-element" size="sizeIcon50" src="./avatars/avatar8.png" name="GitHub avatar" />
+    </CornerOrnamentElement>
+    <CornerOrnament data-testid="cornerOrnament">
+      <Box data-testid="ornament">
+        <LogoTwilioIcon decorative size="sizeIcon40" />
+      </Box>
+    </CornerOrnament>
+  </CornerOrnamentContainer>
+);
+
+const CustomizedCornerOrnament: React.FC<{
```

Comment:
suggestion(non-blocking): this could be combined with `ExampleCornerOrnament` if you add an optional `element` prop